Arithmetic/Exercise_9/Program.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Runtime.Intrinsics.X86;
+
+namespace Exercise_9
+{
+    class Program
+    {
+        static void BmiCalculatorPounds()
+        {
+            Console.WriteLine("Enter your weight in pounds");
+            int.TryParse(Console.ReadLine(), out var weight);
+            Console.WriteLine("Enter your height in inches");
+            int.TryParse(Console.ReadLine(), out var height);
+            var bmi = 703 * (weight / Math.Pow(height, 2));
+
+            Console.WriteLine($"Your BMI is: {Math.Round(bmi, 2)}");
+            switch (bmi)
+            {
+                case < 18.5:
+                    Console.WriteLine("You're underweight!");
+                    break;
+                case > 18.5 and < 24.9:
+                    Console.WriteLine("You have normal weight!");
+                    break;
+                case > 25 and < 29.9:
+                    Console.WriteLine("You're overweight!");
+                    break;
+                case > 30 and < 39.9:
+                    Console.WriteLine("You're obese!");
+                    break;
+                case > 40:
+                    Console.WriteLine("You're morbidly obese!");
+                    break;
+            }
+        }
+
+        static void BmiCalculatorKilos()
+        {
+            Console.WriteLine("Enter your weight in kilograms");
+            int.TryParse(Console.ReadLine(), out var weight);
+            Console.WriteLine("Enter your height in centimeters");
+            int.TryParse(Console.ReadLine(), out var height);
+            var bmi = (weight / Math.Pow(height, 2) * 10000);
+            Console.WriteLine($"Your BMI is: {Math.Round(bmi, 2)}");
+            switch (bmi)
+            {
+                case < 18.5:
```

Comment:
Un kas ir pie 18.5?

Reply:
Nesapratu. Liekas ir pareizi:

case < 18.5:
Console.WriteLine("You're underweight!");
break;

Reply:
Tev ir statement priekš case < 18.5 un case > 18.5. Kur paliek pats 18.5?

Reply:
sapratu, izlaboju